index.dd

```diff
@@ -76,6 +76,32 @@ void main()
 }
 ----
 )
+$(EXTRA_EXAMPLE
+----
+// subtyping with alias this
+struct Point
+{
+    private double[2] p;
+    // p is used by the compiler to lookup unknown symbols
```

Comment:
Unknown symbols and operators? Just to clarify, since a newcomer may not know how looking up unknown symbols allows assigning arrays to `Point`.